BUILD.gn

```diff
@@ -317,11 +326,15 @@ config("spvtools_internal_config") {
 
   configs = [ ":spvtools_public_config" ]
 
+  cflags = []
   if (is_clang) {
-    cflags = [
+    cflags += [
       "-Wno-implicit-fallthrough",
       "-Wno-newline-eof",
     ]
+  } else if (!is_win) {
+    # Work around a false-positive on a Skia GCC 10 builder.
+    cflgas += [ "-Wno-format-truncation" ]
```

Comment:
@Kangz this CL broke a roll into chromium:
https://chromium-review.googlesource.com/c/chromium/src/+/2751130
due to the typo on this line.

Autoroller is currently paused:
https://autoroll.skia.org/r/vulkan-deps-chromium-autoroll